kbfstool/md_main.go

```diff
@@ -11,10 +11,10 @@ const mdUsageStr = `Usage:
   kbfstool md [<subcommand>] [<args>]
 
 The possible subcommands are:
-  dump		Dump metadata objects
-  check		Check metadata objects and their associated blocks for errors
-  reset		Reset a broken top-level folder
-
+  dump	      Dump metadata objects
+  check	      Check metadata objects and their associated blocks for errors
+  reset	      Reset a broken top-level folder
+  forceQR     Append a fake quota reclamation record to the folder history
```

Comment:
isn't the usual capitalization for subcommands force-qr? (don't care too much about it)